server/liveblog/client_modules/client_modules.py

```diff
@@ -36,7 +37,7 @@ class ClientBlogsResource(BlogsResource):
     schema.update(BlogsResource.schema)
 
 
-class ClientBlogsService(BlogService):
+class ClientBlogsService(BaseService):
```

Comment:
It resolves a bug that broke the embed route, because of a new permission check introduced on `devel`. 
Without this, I can't test the embed when I'm based on `devel`.